src/scraping.test.ts

```diff
@@ -1,17 +1,17 @@
 import { expect, test } from "vitest";
 import { CatalogConfig } from "./catalog";
 import { SitemapDocument, UrlDocument } from "./document";
-import { testClient } from "./src/vitest-test-client";
+import { testClient } from "./vitest-test-client";
 
 const runScraperTests = process.env.RUN_SCRAPER_TESTS === "true";
 
-const expectedSitemapUrls = 4;
+const expectedSitemapUrls = 28;
 
 test.skipIf(!runScraperTests)(
   "Test scraping single URL",
   { timeout: 60000 },
   async () => {
-    const catalogName = `catalog-${Math.floor(Math.random() * 10000)}`;
+    const catalogName = `catalog-${Math.floor(Math.random() * 1000000)}`;
```

Comment:
What if used a timestamp instead of a random number?

Reply:
Oh hell yeah - I think I came up with a good pattern, if people like it we can use it elsewhere... See the next commit